ask-sdk-model-runtime/ask_sdk_model_runtime/base_service_client.py

```diff
@@ -215,7 +215,7 @@ def __build_query_string(query_params, is_query_start):
             encoded_query_params.append(
                 (param_name_encoded, param_value_encoded))
 
-        return query_string + urlencode(encoded_query_params, doseq=True)
+        return query_string + urlencode(encoded_query_params)
```

Comment:
Why was this removed? We need the `doseq=True` for supporting multiple query params for same key